setup/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  ovehub-ove-asset-db:
+    image: mariadb
+    ports:
+    - "3306:3306"
+    volumes:
+      - "ovehub-ove-asset-db-data:/var/lib/mysql"
+    environment:
+      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+      MYSQL_DATABASE: ""
+      MYSQL_USER: ""
+      MYSQL_PASSWORD: ""
+
+  ovehub-ove-asset-manager:
+    image: ovehub/ove-asset-manager:latest
+    depends_on:
+    - ovehub-ove-asset-db
+    ports:
+    - "80:80"
+    environment:
+      s3Client__AccessKey: ""
+      s3Client__Secret: ""
+      s3Client__ServiceURL: ""
+      ServiceHostUrl: ""
+      MariaDB__ConnectionString: "Server=ovehub-ove-asset-db;Port=3306;Database=DATABASE;User=USER;Password=PASSWORD;"
+      MariaDB__Version: "10.3.10" 
+
+  ovehub-ove-service-imagetiles:
+    image: ovehub/ove-service-imagetiles:latest
+    depends_on:
+    - ovehub-ove-asset-manager
+    ports:
+    - "8090:80"
```

Comment:
Does this mean multiple internal services run on port 80?

Reply:
No. Within a single docker image only one service runs on 80. Both services run in different docker images and the exposed ports are remapped using docker rules.

Reply:
Yet again the ports need to be fixed - FTR, @oserban.